app/models/course.rb

```diff
@@ -131,6 +133,10 @@ def not_available?
     !exposed_in_find
   end
 
+  def open?
+    applications_open_from <= Time.zone.today && exposed_in_find && application_status_open?
+  end
```

Comment:
Now I understand from our conversation some weeks ago :). Awesome ❤️ 
Is the goal in a future PR to check the course selection and submission around the `open?` and the period between `find_opens` and `apply_opens`?

Reply:
I think not, if I understand you correcty